Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -109,11 +109,11 @@ def test_bottle
     formula_file = CoreFormulaRepository.new.formula_dir/"testball.rb"
     formula_file.write <<-EOS.undent
       class Testball < Formula
-        url "https://example.com/testabll-0.1.tar.gz"
+        url "https://example.com/testball-0.1.tar.gz"
       end
     EOS
     HOMEBREW_CACHE.cd do
-      assert_match /testball-0\.1.*\.bottle\.tar\.gz/,
+      assert_match %r{testball-0\.1.*\.bottle\.tar\.gz},
```

Comment:
Yeah, `brew readall --syntax --aliases` also picks up on it (It throws out to `ruby -wc` I believe), which is quite a bit more irritating than a casual linting highlight in the $EDITOR sadly:
```
test_integration_cmds.rb:116: warning: ambiguous first argument; put parentheses or even spaces
```